php/ad-servers/class-ad-layers-dfp.php

```diff
@@ -342,7 +342,12 @@ public function get_settings_fields() {
 						'page_type' => new Fieldmanager_Select(
 							array(
 								'label' => __( 'Page Type', 'ad-layers' ),
-								'options' => Ad_Layers::instance()->get_page_types(),
+								'options' => array_merge(
+									array(
+										'all' => __( 'All Pages', 'ad-layers' ),
```

Comment:
Yeah I'm aware. I contemplated this earlier. These were meant to follow the WordPress template hierarchy. I'm fearful to remove or change it now if anyone is using it.